doc/elements.md

```diff
@@ -415,7 +416,7 @@ Attribute | Type | Default | Description
 `label` | text | — | A prefix to display before the input box (e.g., `label="$x =$"`).
 `suffix` | text | — | A suffix to display after the input box (e.g., `suffix="items"`).
 `display` | "block" or "inline" | "inline" | How to display the input field.
-`remove-leading-trailing` | boolean | false | Whether or not to remove leading and trailing blank spaces from the input string.
+`remove-leading-trailing` | boolean | true | Whether or not to remove leading and trailing blank spaces from the input string.
```

Comment:
Was there any test run to verify question stability before flipping this default?

Reply:
Good point, we've decided to switch this back.